sdk/ml/azure-ai-ml/tests/test_configs/components/component_with_additional_includes/helloworld_invalid_additional_includes_existing_file.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: additional_files
+version: 0.0.1
+type: command
+description: A sample to demonstrate component with additional files
+inputs: {}
+outputs: {}
+environment:
+  image: zzn2/azureml_sdk
+command: >-
+  echo Hello World
+code: ./
```

Comment:
Do we allow code and additional_includes work together?

Reply:
yes